pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -529,10 +534,15 @@ def select_accelerator(self) -> Accelerator:
             acc_cls = CPUAccelerator
         # as precision_plugin is dependent on training_type_plugin, make sure
         # that we first select training_type_plugin, then precision_plugin
-        return acc_cls(
+        accelerator = acc_cls(
             training_type_plugin=self.training_type_plugin,
             precision_plugin=self.precision_plugin,
         )
+        # transfer ownership of the plugins to the accelerator
```

Comment:
in what sense do the lines below transfer ownership? is the comment meant to go above the accelerator creation?

Reply:
before the accelerator was created before, the ownership of those plugins was in the connector (by potentially being the only class to hold a strong reference to them). Now that the accelerator has them, it should be the only place with a strong ref.